tests/lib/rules/indent.js

```diff
@@ -11473,6 +11572,41 @@ ruleTester.run("indent", rule, {
                 [6, 4, 0, "Punctuator"],
                 [7, 4, 0, "Punctuator"]
             ])
+        },
+        {
+            code: unIndent`
+                const foo = async (arg1,
+                                    arg2) =>
+                {
+                  return arg1 + arg2;
+                }
+            `,
+            output: unIndent`
+                const foo = async (arg1,
+                                   arg2) =>
+                {
+                  return arg1 + arg2;
+                }
+            `,
+            options: [2, { FunctionDeclaration: { parameters: "first" }, FunctionExpression: { parameters: "first" } }],
+            parserOptions: { ecmaVersion: 2020 },
+            errors: expectedErrors([
+                [2, 19, 20, "Identifier"]
+            ])
+        },
+        {
+            code: unIndent`
+                const a = async
+                 b => {}
+            `,
+            output: unIndent`
+            const a = async
+            b => {}
```

Comment:
It would be nice to align `output` with `code`

All others look good, thanks for formatting the tests!